presto-main/src/main/java/com/facebook/presto/SystemSessionProperties.java

```diff
@@ -350,6 +350,9 @@ public final class SystemSessionProperties
     private static final String NATIVE_EXECUTION_PROGRAM_ARGUMENTS = "native_execution_program_arguments";
     public static final String NATIVE_EXECUTION_PROCESS_REUSE_ENABLED = "native_execution_process_reuse_enabled";
     public static final String NATIVE_DEBUG_VALIDATE_OUTPUT_FROM_OPERATORS = "native_debug_validate_output_from_operators";
+
+    public static final String NATIVE_MAX_PARTIAL_AGGREGATION_MEMORY = "native_max_partial_aggregation_memory";
```

Comment:
Do you want to expose max_extended_partial_aggregation_memory as well? Velox supports to dynamically adjust aggregation memory limit in a range defined by the above two configs? Thanks!